### PR TITLE
place node_modules folder at project root level

### PR DESCRIPTION
**Pull request by gtebrean**
### What does this PR do?
places the node_module folder at the root of the new generated templates projects

### Where should the reviewer start?
changed files

### Why is it needed?
Currently node_modules folder is in the build folder and every time clean runs will be deleted and dependency libs contracts have to be downloaded. So moving them outside the build folder will make things easier.
 
Screenshot 2022-05-16 at 20 42 09

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -19,7 +19,7 @@ mainClassName = 'org.web3j.console.Web3j'
 applicationName = 'web3j'
 
 ext {
-    web3jVersion = '4.8.4'
+    web3jVersion = '4.9.0'
```
Comment (andrii-kl) [inline build.gradle:22]: 4.9.2

File: src/main/resources/project/build.gradleImport.template [hunk 2]
```diff
@@ -3,7 +3,7 @@ plugins {
     id 'org.jetbrains.kotlin.jvm' version '1.3.61'
     id 'application'
     id "com.github.johnrengelman.shadow" version "5.2.0"
-    id 'org.web3j' version '4.8.4'
+    id 'org.web3j' version '4.9.0'
```
Comment (andrii-kl) [inline src/main/resources/project/build.gradleImport.template:6]: 4.9.2

File: src/main/resources/project/build.gradleImportJava11.template [hunk 3]
```diff
@@ -3,7 +3,7 @@ plugins {
     id 'org.jetbrains.kotlin.jvm' version '1.3.61'
     id 'application'
     id "com.github.johnrengelman.shadow" version "5.2.0"
-    id 'org.web3j' version '4.8.4'
+    id 'org.web3j' version '4.9.0'
```
Comment (andrii-kl) [inline src/main/resources/project/build.gradleImportJava11.template:6]: 4.9.2

File: src/main/resources/project/build.gradleJava11.template [hunk 4]
```diff
@@ -3,7 +3,7 @@ plugins {
     id 'org.jetbrains.kotlin.jvm' version '1.3.61'
     id 'application'
     id "com.github.johnrengelman.shadow" version "5.2.0"
-    id 'org.web3j' version '4.8.4'
+    id 'org.web3j' version '4.9.0'
```
Comment (andrii-kl) [inline src/main/resources/project/build.gradleJava11.template:6]: 4.9.2

**Comment by gtebrean** [comment]
test pipeline needs to be rerun before release

**Pull request merged**